src/mds/MDSRank.h

```diff
@@ -219,7 +219,7 @@ class MDSRank {
     bool allows_multimds_snaps() const { return mdsmap->allows_multimds_snaps(); }
 
     bool is_cache_trimmable() const {
-      return is_clientreplay() || is_active() || is_stopping();
+      return is_standby_replay() || is_clientreplay() || is_active() || is_stopping();
```

Comment:
Was there any reason why standby_replay was left out of regular trimming before this?

Reply:
It was (and still is) initiated by another code path in MDLog. It just doesn't happen often enough to keep up with hot journals.

Reply:
Ack.